templates/master/01-master-kubelet/_base/units/kubelet-cleanup.service.yaml

```diff
@@ -0,0 +1,14 @@
+name: kubelet-cleanup.service
+enabled: true
+contents: |
+  [Unit]
+  Description=Kubernetes Kubelet After Reboot Cleanup
+  Before=kubelet.service
+
+  [Service]
+  Type=oneshot
+  ExecStart=/bin/rm -f /var/lib/kubelet/cpu_manager_state
```

Comment:
Also, systemd-tmpfiles has streamlined support for this, e.g.
```
r! /var/lib/kubelet/*_manager_state
```
dropped in e.g. `/etc/tmpfiles.d/kubelet-state.conf` by MCO templates *or* IMO even more preferable: included in `kubelet.rpm` and hence shipped in RHCOS would also be simpler.